apps/web/lib/getTeamMemberEmailFromCrm.ts

```diff
@@ -231,7 +231,6 @@ export async function getTeamMemberEmailForResponseOrContactUsingUrlQuery({
   log.debug("getTeamMemberEmailForResponseOrContactUsingUrlQuery", safeStringify({ query }));
 
   const crmAppSlug = getEnabledRoutingFormAppSlugFromQuery(query);
-  if (!crmAppSlug) return null;
```

Comment:
`crmAppSlug` is actually not needed when we just need to use the contact owner.

Later in code we check if crmAppSlug is set then use routing-form appBookingForm handler and otherwise just use contact owner